_posts/2023-02-01-fooconf-helsinki.md

```diff
@@ -0,0 +1,91 @@
+
+After lunch, we both listened to the speech about Security by design by **Daniel Deogun** from OmegaPoint. It was about how to implicitly write safe software by going through the log4j vulnerabilities as an example. We learned that good design mitigates risks but not fully. Good discussion about what is safe and what is not.
+
+![paste](/img/2023-fooconf/fooconf2023_6.jpg)
+_Security by design by Daniel Deogun_
+
+Teemu continued with a speech about Apache Pulsar, an event streaming platform by **Mary Grygleski** from DataStax It started with the basics of event streaming and went through the details of Apache Pulsar. Interesting speech which also discussed how Pulsar relates to Kafka, which can be used as a bolt-on with Kafka
+
+Markus listened to **Leif Åstrand** from Vaadin about Why and How to build a Collaborative UX. The speech was a presentation of the experiences of Vaadin on how they have tackled the issues when a collaborative UX is needed. Starting with use cases the speech went on to describe the architectural decisions to consider. The speech ended with a discussion about clustering in collaborative apps and how to implement a “distributed system without building a distributed system”. The speech provided good ideas for Markus which might come in handy shortly in his project work.
+
+Next in line for both Teemu and Markus was a speech called Devops that matters by **Melissa McKay** from JFrog. It was a discussion about starting with DevOps from a developer's point of view. Some of the common problems of dev teams and operations were discussed. It was emphasized that DevOps is a mindset from code to deployment and back. The most important takeaway from the speech was that silos should be eliminated! Many useful practical tips were given.
+
+![paste](/img/2023-fooconf/fooconf2023_7.jpg)
+_Devops that matters by Melissa McKay_
+
+Then it was time for the last speech of the day, Thinking Architecturally by **Nate Schutta**. It was a truly humorous speech about software architecture. Nate is an experienced speaker and the speech provided some insights into everyday work in a very humorous way. There were some common themes with the first speech of the day by Tom Cools as Nate also emphasized the importance of constant learning and focusing on the core skills instead of learning all the new shiny technologies. We should learn from good and bad decisions. It was also repeated that we should be kind to ourselves.
```

Comment:
"humorous" gets used a lot in this text. I suggest using something  like "playful", "funny", "hilarious" to mix things up a little bit.

Reply:
Fixed, thanks!